.github/scripts/release.py

```diff
@@ -66,6 +67,23 @@ def cargo_set_version(args):
     run_command(['cargo', 'set-version', '--exclude', 'swf', *args])
 
 
+def gh_list_nightly_tags(limit):
+    tags_json = run_command([
+        'gh', 'release', 'list',
+        '--order', 'desc',
+        '--limit', str(limit),
+        '--json=tagName',
+        '--jq', '.[] | select(.tagName | startswith("nightly-"))',
+    ])
+    for tag_json in tags_json.splitlines():
+        tag = json.loads(tag_json)
+        yield tag['tagName']
+
+
+def gh_get_last_nightly_tag():
+    return next(gh_list_nightly_tags(16), None)
```

Comment:
don't really understand why the param is here, might as well hardcode it in the function itself?

Also, can you add a comment that this would need a refactor if we ever did non-linear releases?

Reply:
I added the param here because it's an important assumption of `gh_get_last_nightly_tag` and not `gh_list_nightly_tags`. The assumption is that the last nightly tag will be within the last 16 releases.
 
> Also, can you add a comment that this would need a refactor if we ever did non-linear releases?

Hmmm, I don't see why it wouldn't work with non-linear releases. It should work the same as long as we release nightly releases from the main branch.

Reply:
Done